packages/amplify-graphql-transformer-migrator/src/constants/graphql-directives.ts

```diff
@@ -0,0 +1,145 @@
+export const GRAPHQL_DIRECTIVES_SCHEMA = `
```

Comment:
Attila doesn't want any dependencies on V1 packages because it makes it easier for us to move this out of the CLI. I have more changes to make like renaming the file to be V1 specific since we're going to just try and pre-ship it unhooked. I think that should be fine since we're not focused on feature addition in V1